docs/content/snippets/quick-install.mdx

```diff
@@ -20,4 +20,14 @@ Find more [versions of Sui for Windows](https://community.chocolatey.org/package
 
 </TabItem>
 
-</Tabs>
\ No newline at end of file
+<TabItem label="suiup (experimental)" value="win">
```

Comment:
Why is `win` the value? Really doesn't matter as long as it's unique to the tabitems in this tabs element.

Reply:
This is the cause of the ci error it seems because it's not unique.